pkg/util/util.go

```diff
@@ -777,6 +777,40 @@ func GetAWSRegion() (string, error) {
 	return awsRegion, nil
 }
 
+// IsValidS3BucketName checks the name according to
+// https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-s3-bucket-naming-requirements.html
+func IsValidS3BucketName(name string) bool {
+	//The bucket name can be between 3 and 63 characters long
+	if len(name) < 3 || len(name) > 63 {
+		return false
+	}
+
+	for i, c := range name {
```

Comment:
Why not regexp?
https://github.com/noobaa/noobaa-core/blob/f8eb884322ddc17848eaaed1cf4c768c7620d9bc/src/server/system_services/bucket_server.js#L39-L40

Reply:
changed